.ci/upload.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+rm -rf dist
+mkdir dist/
+
+export STORAGE_ACCOUNT_NAME=platopslackhelpbotai
+export CONTAINER_NAME=the-hmcts-way
+
+function extractMain () {
+    local path=$1
+
+    DIRECTORY=$(dirname "$path")
+    mkdir -p dist/$DIRECTORY
+
+    cat $path | htmlq --text 'main' --remove-nodes '[data-module="page-expiry"]' > dist/$path
+}
+
+export -f extractMain
+
+find build -name "*.html" -not -path "build/search/*" -exec bash -c "extractMain \"{}\"" \;
```

Comment:
the generated search page in the framework doesn't need indexing and it also have a heading across multiple lines which would need a fixup if we wanted to keep it